src/nunit-gui/CommandLineOptions.cs

```diff
@@ -190,7 +190,7 @@ private int RequiredInt(string val, string option)
                 if (int.TryParse(val, out r))
                     result = r;
                 else
-                    ErrorMessages.Add("An int value was exprected for option '{0}' but a value of '{1}' was used");
+                    ErrorMessages.Add("An int value was expected for option '{0}' but a value of '{1}' was used");
```

Comment:
This is in a method which is not used. Underneath is commented-out code. Should I remove the whole method, possibly also the commented-out code?